src/media_tools/av_parsers.c

```diff
@@ -5928,28 +5928,29 @@ static s32 avc_parse_pic_timing_sei(GF_BitStream *bs, AVCState *avc)
 			return 1;
 		}
 
+		pt->num_clock_ts = NumClockTS[pt->pic_struct];
 		for (i = 0; i < NumClockTS[pt->pic_struct]; i++) {
 			if (gf_bs_read_int_log_idx(bs, 1, "clock_timestamp_flag", i)) {
-				Bool full_timestamp_flag;
+				AVCSeiPicTimingTimecode *tc = &pt->timecodes[i];
 				gf_bs_read_int_log_idx(bs, 2, "ct_type", i);
 				gf_bs_read_int_log_idx(bs, 1, "nuit_field_based_flag", i);
 				gf_bs_read_int_log_idx(bs, 5, "counting_type", i);
-				full_timestamp_flag = gf_bs_read_int_log_idx(bs, 1, "full_timestamp_flag", i);
+				Bool full_timestamp_flag = gf_bs_read_int_log_idx(bs, 1, "full_timestamp_flag", i);
 				gf_bs_read_int_log_idx(bs, 1, "discontinuity_flag", i);
 				gf_bs_read_int_log_idx(bs, 1, "cnt_dropped_flag", i);
-				gf_bs_read_int_log_idx(bs, 8, "n_frames", i);
+				tc->n_frames = gf_bs_read_int(bs, 8);
 				if (full_timestamp_flag) {
-					gf_bs_read_int_log_idx(bs, 6, "seconds_value", i);
-					gf_bs_read_int_log_idx(bs, 6, "minutes_value", i);
-					gf_bs_read_int_log_idx(bs, 5, "hours_value", i);
+					tc->seconds = gf_bs_read_int(bs, 6);
```

Comment:
You changed the function here, which modifies in turn the traces. Any reason for changing?

Reply:
No reason, I will revert it.